pkg/runtime/scheme/scheme_test.go

```diff
@@ -0,0 +1,29 @@
+package scheme_test
+
+import (
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
+)
+
+var _ = Describe("Scheme", func() {
+	Describe("NewAddToScheme", func() {
+		It("should return a function that will add the types to the scheme", func() {
+			gv := schema.GroupVersion{Group: "core", Version: "v1"}
+			a := scheme.NewAddToScheme(gv, func() []runtime.Object {
+				return []runtime.Object{&v1.Pod{}, &v1.PodList{}}
+			})
+			s := runtime.NewScheme()
+
+			Expect(a(s)).To(Succeed())
+			Expect(s.AllKnownTypes()).To(HaveLen(13))
+			Expect(s.AllKnownTypes()[gv.WithKind("Pod")]).To(Equal(reflect.TypeOf(v1.Pod{})))
+			Expect(s.AllKnownTypes()[gv.WithKind("PodList")]).To(BeEquivalentTo(reflect.TypeOf(v1.PodList{})))
```

Comment:
Why this one is using `BeEquivalentTo` but not `Equal`?
What is the difference between Pod and PodList?